packages/direction-links/src/js/react.js

```diff
@@ -80,4 +88,30 @@ AUdirectionLink.defaultProps = {
 	linkComponent: 'a',
 };
 
+/**
+ * TO DO JS DOCS!!!!!!!!!!!!!!!!!!!!!!
+ * @param {} param
+ */
+const AUdirectionArrow = ({ direction, text }) => {
```

Comment:
I was lazy at the time I wrote this, maybe we can rename this? Or do you think its a good name for it?